### PR TITLE
[Reviewer: Matt] Build the bulk provisioning tools

### LINKED ISSUE #267: Java bulk provisioning should be included in homestead-prov package
The [source](https://github.com/Metaswitch/crest/tree/dev/src/metaswitch/crest/tools/sstable_provisioning) used to be included, but was removed as part of recent refactoring.  We should either reinstate this, or (better) build it and ship binaries as part of the package.

Marking as bug because this is technically a regression.  There is a work-around (to download the code separately and build it).

### PR DESCRIPTION
**Pull request by eleanor-merry**
Matt, can you review this change to build the java bulk provision tool and include it in crest installs?

Tested live

Fixes #267

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,4 @@ parts
 *.swp
 .coverage
 htmlcov/
+src/metaswitch/crest/tools/sstable_provisioning/*class
```
Comment (bossmc) [inline .gitignore]: `*.class`?

File: debian/crest.install [hunk 2]
```diff
@@ -1,2 +1,4 @@
 .crest-eggs/* usr/share/clearwater/crest/.eggs/
 src/metaswitch/crest/tools usr/share/clearwater/crest/src/metaswitch/crest/
+debian/crest/usr/share/clearwater/crest/src/metaswitch/crest/tools/sstable_provisioning/BulkProvision /usr/share/clearwater/crest/tools/sstable_provisioning
```
Comment (bossmc) [inline debian/crest.install]: Unnecessary

File: Makefile [hunk 3]
```diff
@@ -93,20 +93,21 @@ ${ENV_DIR}/.eggs_installed : $(ENV_DIR)/bin/python $(shell find src/metaswitch -
...
 
 .PHONY: pyclean
 pyclean:
 	-find src -name \*.pyc -exec rm {} \;
-	-rm -r src/*.egg-info
-	-rm .coverage
-	-rm -r htmlcov/
+	-rm -rf src/*.egg-info
+	-rm -f .coverage
+	-rm -rf htmlcov/
```
Comment (bossmc) [inline Makefile]: Why all the `-f` parameters!?

Comment (mirw) [inline Makefile]: My guess is that otherwise pyclean fails if the python components haven't already been fully built.

**Comment by mirw** [comment]
Looks good.  Not much to add to Andy's comments except

*   should we updated docs/Development.md to specify the build dependency on the JDK?
*   have you live-tested building from a clean checkout and installing?

File: Makefile (hunk 3 above)
Comment (eleanor-merry) [inline Makefile]: Yup - it's annoying

**Comment by eleanor-merry** [comment]
Testing done as suggested

**Pull request merged**